lib/utils/ensure-posix.js

```diff
@@ -0,0 +1,14 @@
+var path = require('path');
```

Comment:
This seems to be the same as https://github.com/stefanpenner/ensure-posix-path (which we are already using in ember-cli itself, so its probably fine to require).

Fine for a follow up PR though.